ksql-engine/pom.xml

```diff
@@ -95,6 +95,12 @@
             <artifactId>janino</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.github.lukehutch</groupId>
+            <artifactId>fast-classpath-scanner</artifactId>
```

Comment:
I'm pretty sure MIT is ok. But will check. The fact that it works for what we need it do is reliable enough

Reply:
There is also a good reason i used this. It is fast and doesn't need to load the classes and pollute the perm-gen. It is also actively maintained. https://github.com/lukehutch/fast-classpath-scanner/wiki#how-fast-is-fastclasspathscanner

Reply:
By reliability I meant what if the person owning the repo moves it to another place. 
Can we create a confluent repo and add it to our repo so we are sure it will be available all the time?